editor/src/clj/editor/graphics.clj

```diff
@@ -153,7 +153,6 @@
 
 (defn resize-doubles [double-values semantic-type ^long new-element-count]
   {:pre [(vector? double-values)
-         (keyword? semantic-type)
```

Comment:
In the case where the component has an overridden vertex attribute and changes material, the intention is to still show the property in the property view, but if the 'incoming' material doesn't have the overridden vertex attribute in its attribute table this assert will be triggered since the component attribute doesn't have any semantic types.

Reply:
```(or (nil? semantic-type) (keyword? semantic-type))```
Might be preferable so that we can still catch bugs as intended?

Reply:
Sure, sounds reasonable!